docs/release_process.rst

```diff
@@ -42,7 +43,8 @@ and functionalities.
 1. Create a new branch off of the ``master`` or ``release`` named something
    like ``release-[major,minor]-{NEW_VERSION}``.
 
-  a) Increment patch value in ``VERSION`` file.
+  a) Increment the version number in the ``__version__`` value, defined in
+  ``smqtk_core/__init__.py``.
```

Comment:
This needs spaces to line up the indentation. Without this the rendering is disturbed. (check out the rich diff for what I mean)

Reply:
Thanks, fixed in 7233a4b.